### PR TITLE
Adding Glances & qDirstat to Tipi!

### PR DESCRIPTION
**Pull request by itsrllyhim**
Please see 'Read Me'.

### CODE REVIEW

File: apps/glances/config.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../schema.json",
+  "name": "Glances",
+  "port": 8420,
+  "available": true,
+  "exposable": true,
```
Comment (DrMxrcy) [inline apps/glances/config.json:6]:
```suggestion
  "exposable": false,
```
There should be no reason to expose this with a Public Domain IMO. Can confirm with @meienberger

File: apps/glances/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            # Local domain
+            traefik.http.routers.glances-local-insecure.rule: Host(`glances.${LOCAL_DOMAIN}`)
+            traefik.http.routers.glances-local-insecure.entrypoints: web
+            traefik.http.routers.glances-local-insecure.service: glances
+            traefik.http.routers.glances-local-insecure.middlewares: glances-web-redirect
+            # Local domain secure
+            traefik.http.routers.glances-local.rule: Host(`glances.${LOCAL_DOMAIN}`)
+            traefik.http.routers.glances-local.entrypoints: websecure
+            traefik.http.routers.glances-local.service: glances
+            traefik.http.routers.glances-local.tls: true
```
Comment (DrMxrcy) [inline apps/glances/docker-compose.yml:42]:
```suggestion
            # Local domain
            traefik.http.routers.glances-local-insecure.rule: Host(`glances.${LOCAL_DOMAIN}`)
            traefik.http.routers.glances-local-insecure.entrypoints: web
            traefik.http.routers.glances-local-insecure.service: glances
            traefik.http.routers.glances-local-insecure.middlewares: glances-web-redirect
            # Local domain secure
            traefik.http.routers.glances-local.rule: Host(`glances.${LOCAL_DOMAIN}`)
            traefik.http.routers.glances-local.entrypoints: websecure
            traefik.http.routers.glances-local.service: glances
            traefik.http.routers.glances-local.tls: true
```

In accordance with my Suggestion Above

**Review by DrMxrcy — COMMENTED** [review]
Looks Good. Deployed well, Just the removal of the exposable domain, without any Authentication in front of it.

File: apps/glances/config.json (hunk 1 above)
Comment (itsrllyhim) [reply apps/glances/config.json:6]: Sounds good, no disagreement.

File: apps/glances/docker-compose.yml (hunk 2 above)
Comment (itsrllyhim) [reply apps/glances/docker-compose.yml:42]: Cheers, thanks!

**Comment by itsrllyhim** [comment]
Alrighty that was my first attempt at doing this by my lonesome so it took a couple of tries to pass the tests- if someone (looking at you here @DrMxrcy haha) could double check my work, I would greatly appreciate it!

**Comment by itsrllyhim** [comment]
Okay! Another update on qDirStat: Initially passed the tests but failed to install to tipi. I corrected the issue causing that (Config directory had to match a specific constraint), I also fixed a minor issue with the ReadMe not loading the correct image for a screenshot. 

**Problems:**

I can't seem to get it to scan outside of the config directory. The documentations provided an ENV variable that didn't seem to have an impact even when overridden (since removed from the docker-compose.yaml). It also seems the entire tool runs inside of a Kasm machine, where if the user closes out of the program, they then become sort of "stuck" and have to manually stop and restart the app to get it back up.  I am in touch with the LinuxServer.io folks to see if we can find a solution- if so, will update the image.

File: apps/glances/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+services:
+    glances:
+        container_name: glances
+        restart: unless-stopped
+        ports:
+            - ${APP_PORT}:61208
+            - 8421:61209
```
Comment (meienberger) [inline apps/glances/docker-compose.yml:8]: For what is this port needed? I removed it and app launched the same

File: apps/qdirstat/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+version: "1.8.1-ls82"
+services:
+    qdirstat:
+        container_name: qdirstat
+        restart: unless-stopped
+        ports:
+            - ${APP_PORT}:3000
+        volumes:
+            - ${APP_DATA_DIR}/data/qdirstat/config
+            - ${APP_DATA_DIR}/data/qdirstat/app/data
```
Comment (meienberger) [inline apps/qdirstat/docker-compose.yml:10]: I think this app would be more useful if we could see the whole system tree from `/` what do you think?

**Review by meienberger — CHANGES_REQUESTED** [review]

File: apps/glances/docker-compose.yml (hunk 3 above)
Comment (itsrllyhim) [reply apps/glances/docker-compose.yml:8]: Honestly, not particularly sure- Steve and I just followed the docs. If it works without it, then there's no need for it. Feel free to remove it!

File: apps/qdirstat/docker-compose.yml (hunk 4 above)
Comment (itsrllyhim) [reply apps/qdirstat/docker-compose.yml:10]: You're right, I think that would solve the problem I alluded to [here](https://github.com/meienberger/runtipi-appstore/pull/1161#issuecomment-1710990364). Feel free to make the change, or ping me if you would like me to!

**Pull request merged**